### PR TITLE
feat: implement F3 CLI to list power table and proportional power at instance

### PR DESCRIPTION
**Pull request by masih**
## Proposed Changes
Implement utility CLIs to:
* get the power table used by F3 at a given instance ID.
* get total proportional power of a list of actors at a given instance ID.

These utilities allow us to debug the exact participation power for an instance without having to manually calculate it or estimate it from the latest power.

## Checklist

Before you mark the PR ready for review, please make sure that:

- [x] Commits have a clear commit message.
- [x] PR title conforms with [contribution conventions](https://github.com/filecoin-project/lotus/blob/master/CONTRIBUTING.md#pr-title-conventions)
- [x] Update CHANGELOG.md or signal that this change does not need it per [contribution conventions](https://github.com/filecoin-project/lotus/blob/master/CONTRIBUTING.md#changelog-management)
- [x] New features have usage guidelines and / or documentation updates in
  - [ ] [Lotus Documentation](https://lotus.filecoin.io)
  - [ ] [Discussion Tutorials](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)
- [ ] Tests exist for new functionality or change in behavior
- [ ] CI is green

### CODE REVIEW

File: documentation/en/cli-lotus.md [hunk 1]
```diff
@@ -2813,148 +2818,12 @@ OPTIONS:
 NAME:
    lotus f3 list-miners - Lists the miners that currently participate in F3 via this node.
 
-USAGE:
```
Comment (masih) [inline documentation/en/cli-lotus.md:2816]: hmm not sure what's happening here. I'll take a look

File: cli/f3.go [hunk 2]
```diff
@@ -70,6 +75,243 @@ var (
...
+						Name:    "get-proportion",
+						Aliases: []string{"gp"},
+						Usage: `Gets the total proportion of power for a list of actors at a given instance.
+
+The instance may be specified as the first argument. If unspecified,
+the latest instance is used.
+
+The list of actors may be specified as Actor ID or miner address, space
+separated, pied to STDIN. Example:
+  $ echo "1413 t01234 f12345" | lotus f3 powertable get-proportion 42`,
```
Comment (rvagg) [inline cli/f3.go:186]: wouldn't it be easier to just accept these as trailing args? then make the `[instance]` an `[--instance id]`?

Comment (masih) [reply cli/f3.go:186]: All sub commands here take instace as the first arg. I wanted to keep to that. Happy to change it.

**Comment by Kubuxu** [comment]
It will be slightly annoying, but can you separate the commands into separate vars? The diff is hard to read because it got too wide.

**Comment by masih** [comment]
> can you separate the commands into separate vars?

Sure; done.

**Comment by masih** [comment]
The diff looks more verbose now unfortunately. 

If this is hard to review, can break up the vars in a separate PR then rebase this one. Please let me know if you'd prefer it that way @Kubuxu

**Review by Kubuxu — APPROVED** [review]

File: documentation/en/cli-lotus.md [hunk 3]
```diff
@@ -2820,6 +2821,49 @@ OPTIONS:
...
+                       The list of actors may be specified as Actor ID or miner address, space
+                       separated, via arguments. Example:
+                         $ lotus f3 powertable get-proportion -i 42 1413 t01234 f12345
+   help, h             Shows a list of commands or help for one command
+
+OPTIONS:
+   --help, -h  show help
+```
+
+#### lotus f3 powertable get
```
Comment (rvagg) [inline documentation/en/cli-lotus.md:2851]: here's a puzzle, why are we not getting a section with `## lotus f3 powertable get-proportion`?

Comment (masih) [reply documentation/en/cli-lotus.md:2851]: It's a bug: https://github.com/filecoin-project/lotus/issues/12705

Comment (masih) [reply documentation/en/cli-lotus.md:2851]: I have captured my thoughts on the right way to fix this here: https://github.com/filecoin-project/lotus/issues/12706

For now i work around it by avoiding new lines in usage.

**Pull request merged**